zerver/openapi/zulip.yaml

```diff
@@ -12936,6 +12950,19 @@ paths:
                     type: integer
                   default: []
                   example: [1, 10]
+                user_group_ids:
+                  description: |
+                    A list containing the [IDs of the user groups](/api/get-user-groups) that
+                    the newly created user will be automatically added to if the invitation
+                    is accepted. If the list is empty, then the new user will not be
+                    added to any user groups.
```

Comment:
Add another paragraph: "The acting user must have permission to add users to the groups listed in this request.".